README.md

```diff
@@ -626,6 +627,9 @@ Table of Contents
   * [zoho.com](https://www.zoho.com/mail/) — Free Webmail and collaboration for up to 5 users
   * [SimpleLogin](https://simplelogin.io/) – Open source, self-hostable email alias/forwarding solution. Free 5 Aliases, unlimited bandwith, unlimited reply/send. Free for educational staffs (student, researcher, etc).
 
+## File Converters
+  * [Conversion Tools](https://conversiontools.io/) - Online File Converter for documents, images, video, audio, eBooks. REST API is available. Libraries for Node.js, PHP, Python. Support files up to 50 GB (for paid plans). Free tier is limited by file size and number of conversions per day.
```

Comment:
Unfortunately we dont add single entry categories, either add this one to an existing category or add a few more related services - 3 or 4 will do.

Reply:
Yes exactly @drdmitry .
If you don't have more , I guess this thing will fit in APIs section.

Reply:
Also you can find some converters already listed which may ease out your work.

Reply:
Thanks for the suggestions @ripienaar @thispsj, I moved it into the API section and "Storage and Media Processing" (2nd one is because it is not the only API and might be useful for devs who use the website).
Please review.